x-pack/plugins/ml/public/data_frame/common/fields.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// TODO consolidate with anomaly detection wizard code
```

Comment:
Could this be combined with `ml/common/constants/field_types.js` as part of this PR? Or would it be a lot of code churn?

Reply:
Done in 7a46bc4.